server.go

```diff
@@ -791,6 +792,14 @@ func (server *Server) wireFrontendBackend(serverRoute *serverRoute, handler http
 		}
 	}
 
+	// path replace
+	if len(serverRoute.replacePath) > 0 {
```

Comment:
actually, does this function even need to be a method of `Server`? it doesn't access any of it's fields.